iModelCore/ECDb/Tests/NonPublished/ECSqlStatementTests.cpp

```diff
@@ -447,13 +447,13 @@ TEST_F(ECSqlStatementTestFixture, SelectAsterisk)
                 </ECSchema>)xml")));
 
     ECInstanceKey aKey, bKey, subBKey, fileInfoKey, aLinksBKey, subBLinksFileInfoKey;
-    ASSERT_EQ(BE_SQLITE_DONE, GetHelper().ExecuteInsertECSql(aKey, "INSERT INTO ts.A(Name,Size) VALUES('A-1',100)"));

```

Comment:
It's a bit concerning to me that the test changes are ONLY modifying existing test cases to include ONLY instead of adding new test cases

Reply:
fixed